src/scripts/main.js

```diff
@@ -0,0 +1,84 @@
+import './renderButton';
+import './renderForm';
+import './renderPages';
+import { createPagesStructure } from './renderPages';
+
+const paginationComponent = document.querySelector('.pagination-component');
+const DEFAULT_ELEMENT = 1;
+const DEFAULT_MAX_PAGE = 10;
+const leftButton = paginationComponent.querySelector(
+    '.pagination-component__button--left'
+);
+const rightButton = paginationComponent.querySelector(
+    '.pagination-component__button--right'
+);
+
+const paginationComponentInput = paginationComponent.querySelector(
+    '.pagination-component__input'
+);
+
+let currentElement = DEFAULT_ELEMENT;
+let maxElement = DEFAULT_MAX_PAGE;
+
+const updateCurrentElement = () => {
+    const url = new URL(document.URL);
+    if (url.searchParams.has('p')) {
+        console.log(currentElement);
```

Comment:
jak używasz eslinta to dorzuć sobie sprawdzanie console.logów w kodzie który chcesz commitować, coby na repo przypadkiem nie wrzucać, bo potem pójdzie na produkcje i będzie nieładnie